### PR TITLE
Add skeleton "Checkout" page for Marketplace items

### PR DESCRIPTION
**Pull request by zfox23**
Open `<Interface Path>\scripts\system\html\js\marketplacesInject.js` and set `confirmAllPurchases` to `true` if you'd like to test out this PR.

Lots of fancy stuff going on here. This PR modifies the Marketplace DOM to change the "Get" button on Marketplace items to a "Buy" button. When the user presses the "Buy" button, they're presented with a crudely-designed "Checkout" page, which lets them continue to "Buy" the item, or "Cancel" and go back to the item's Marketplace page.

This is one of three components to the initial Commerce work our team is doing - @davidkelly is on the backend piece and, today, @howard-stearns worked on the interaction between the frontend and backend. Good stuff.

It's not necessary to formally test this PR, since this doesn't result in any behavioral changes yet.

### CODE REVIEW

File: scripts/system/marketplaces/marketplaces.js [hunk 1]
```diff
@@ -58,6 +59,12 @@
         UserActivityLogger.openedMarketplace();
         tablet.gotoWebScreen(MARKETPLACE_URL_INITIAL, MARKETPLACES_INJECT_SCRIPT_URL);
         tablet.webEventReceived.connect(function (message) {
+            var parsedJsonMessage = JSON.parse(message);
+            if (parsedJsonMessage.type === "CHECKOUT") {
+                console.log("ZRF: Checkout (Get) Button Clicked: " + JSON.stringify(parsedJsonMessage));
```
Comment (howard-stearns) [inline scripts/system/marketplaces/marketplaces.js]: let's either remove or make less personalized, in case we forget to take it out for a while

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2353/HighFidelity-Beta-PR11143-9d60d5153ff44e983f3c06ddfe440edce320c6d5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2353/HighFidelity-Beta-PR11143-9d60d5153ff44e983f3c06ddfe440edce320c6d5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2362/HighFidelity-Beta-PR11143-61526256bcd56d569964f068f7f78aa5776a9520.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2362/HighFidelity-Beta-PR11143-61526256bcd56d569964f068f7f78aa5776a9520.exe

**Pull request merged**